lib/ProductOpener/KnowledgePanels.pm

```diff
@@ -882,6 +887,65 @@ sub create_nutriscore_panel ($product_ref, $target_lc, $target_cc, $options_ref)
 	return;
 }
 
+sub create_nutriscore_2023_panel ($product_ref, $target_lc, $target_cc, $options_ref) {
+
+	my $version = "2023";
+
+	$log->debug("create nutriscore_2023 panel",
+		{code => $product_ref->{code}, nutriscore_data => deep_get($product_ref, qw/nutriscore 2023 data/)})
+		if $log->is_debug();
+
+	my $panel_data_ref = data_to_display_nutriscore($product_ref, $version);
+
+	# Nutri-Score panel
+	my $grade = deep_get($product_ref, "nutriscore", $version, "grade");
+
+	if ($grade eq "not-applicable") {
+		$panel_data_ref->{title} = lang_in_other_lc($target_lc, "attribute_nutriscore_not_applicable_title");
+	}
+	else {
+		$panel_data_ref->{title}
+			= sprintf(lang_in_other_lc($target_lc, "attribute_nutriscore_grade_title"), uc($grade));
+		if ($panel_data_ref->{nutriscore_unknown_reason_short}) {
+			$panel_data_ref->{subtitle} = $panel_data_ref->{nutriscore_unknown_reason_short};
+		}
+		$panel_data_ref->{subtitle} = lang_in_other_lc($target_lc,
+			"attribute_nutriscore_" . $panel_data_ref->{nutriscore_grade} . "_description_short");
```

Comment:
Aren't we missing a `else` here ? (otherwise you are over writing the subtitle attribute defined just above)

Reply:
Good point, fixing.